docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

```diff
@@ -171,4 +174,65 @@ The API returns the following result:
 }
 ----
 // TESTRESPONSE[s/1562265491319/$body.$_path/]
-// TESTRESPONSE[s/"version": "8.0.0"/"version": $body.version/]
\ No newline at end of file
+// TESTRESPONSE[s/"version": "8.0.0"/"version": $body.version/]
+
+
+[ml-put-dfanalytics-example-r]
+===== {regression-cap} example
+
+The following example creates the `wine_quality_white` {dfanalytics-job}, the 
+analysis type is `regression`:
+
+[source,js]
+--------------------------------------------------
+PUT _ml/data_frame/analytics/wine_quality_prediction
+{
+  "source": {
+    "index": "wine_quality_white"
+  },
+  "dest": {
+    "index": "wine_quality_white_regression"
+  },
+  "analysis": 
+    {
+      "regression": {
+        "dependent_variable": "quality"
+      }
+    }
+}
+--------------------------------------------------
+// CONSOLE
+// TEST[setup:setup_wine_quality_white]
+
+
+The API returns the following result:
+
+[source,js]
+----
+{
+  "id" : "wine_quality_prediction",
+  "source" : {
+    "index" : [
+      "wine_quality_white"
+    ],
+    "query" : {
+      "match_all" : { }
+    }
+  },
+  "dest" : {
+    "index" : "wine_quality_white_regression",
+    "results_field" : "ml"
+  },
+  "analysis" : {
+    "regression" : {
+      "dependent_variable" : "quality"
```

Comment:
I would expect this test fails as the response will also contain `"training_percent": 100` ?